widgets/GoogleSheets-batchsource.json

```diff
@@ -32,7 +32,7 @@
           "label": "Modification Date Range",
           "name": "modificationDateRange",
           "widget-attributes": {
-            "default": "today",
+            "default": "lifetime",
```

Comment:
This change seems unrelated?

Reply:
Its not related to skip empty data config but was a small change so added in the same PR. Should I rollback and raise another PR for it?